templates/page.tmpl

```diff
@@ -55,17 +55,45 @@
         <figure class="lightmode" id="justified-logo">
             <img alt="awesome-logo" src="<TMPL_VAR BASEURL>images/awesome2.svg">
         </figure>
-
-        <nav id="navbar" class="container active-menu-<TMPL_VAR NAME="title">">
-            <a class="menu-about" alt="home" href="<TMPL_VAR BASEURL>index.html">home</a>
-            <a class="menu-download" alt="download" href="<TMPL_VAR BASEURL>download/">download</a>
-            <a class="menu-community" alt="community" href="<TMPL_VAR BASEURL>community/">community</a>
-            <a class="menu-recipes" alt="recipes" href="<TMPL_VAR BASEURL>recipes/">recipes</a>
-            <a class="menu-screenshots" alt="screenshots" href="<TMPL_VAR BASEURL>screenshots/">screenshots</a>
-            <a class="menu-doc" alt="doc" href="<TMPL_VAR BASEURL>doc/">doc</a>
-            <a class="menu-issues" alt="bugs/issues" href="https://github.com/awesomeWM/awesome/issues">bugs/issues</a>
+    </header>
+    <header id="navbar">
+        <a class="logo" href="/">
```

Comment:
This should point to the same URL as the "Home" menu entry. Currently, it does not respect the base URL.